docs/style.css

```diff
@@ -146,12 +146,32 @@ a.new:hover {
   left: 0;
   z-index: 10;
   background: rgba(255,255,255,0.8);
+  color: black;
```

Comment:
Can we change the color to `color:#9c27b0;`?
It would help stick to the color theme.
It'd be great if you can suggest other colors as well.